VOM_Fortran/VOM-docu/source/quickstart.rst

```diff
@@ -1,6 +1,18 @@
 Quickstart
 ===============================
 
+Dependencies
+--------------------------------
+Before starting, make sure an appropriate fortran compiler is installed. The code has been tested and developed mainly with gfortran.
+
+In addition, make sure the netcdf-fortran library is installed:
+
+.. code-block:: bash 
+
+    sudo apt install netcdf-bin libnetcdf-dev libnetcdff-dev
+
+The VOM will by default search for the netcdf installation in usr/include. 
+
```

Comment:
Is it still possible to compile and run the VOM without netcdf? This would be great, so that the old way of running it and analysing the output would still work if desired.

Reply:
So it is really a flag and normally it just produces the same text-files as before. Only setting the flag to true gives netcdf as output.